backends/core/models.py

```diff
@@ -74,6 +83,21 @@ def recipients(self):
       return self.emails_for_notifications.split()
     return []
 
+  def _add_multi_to_memcache(self, mapping,retries = 10, time = MEMCACHE_DEFAULT_EXPIRATION_TIME):
```

Comment:
You can remove the spaces around the equal sign for optional parameters ([pep8 recommendation](https://www.python.org/dev/peps/pep-0008/#other-recommendations)).

Reply:
Please, remove the spaces around optional parameters in other locations too.

Reply:
done